stablehlo/tools/StablehloTranslateMain.cpp

```diff
@@ -97,11 +97,35 @@ class StablehloTranslateInterpreterFallback
                                  stablehlo::Process *process) final {
     llvm::StringRef funcName = op.getParentOfType<func::FuncOp>().getSymName();
     if (auto customCall = dyn_cast<stablehlo::CustomCallOp>(op)) {
-      if (customCall.getCallTargetName() == "check.eq") {
+      auto callTarget = customCall.getCallTargetName();
+      if (callTarget == "check.eq") {
         auto status = evalCustomCallCheckEq(customCall, scope);
         return stablehlo::wrapFallbackStatus(
             std::move(status), funcName, "stablehlo.custom_call(@check.eq)");
       }
+      if (customCall->getNumOperands() == 2) {
```

Comment:
Not sure why we have the num operand chck here?
Can there be `custom_calls` with `callTarget == {check.expect_close, check.expect_eq, check.expect_almost_eq}` but with `!=2` operands?

Reply:
No, but we don't validate these things for each op.. so those without 2 operands will throw the later "unsupported custom call" message.

Not ideal but also I think this is a temporary solution until we have a CC->Check dialect pass